src/main/webapp/user-page.html

```diff
@@ -25,27 +25,32 @@
   </head>
   <body onload="buildUI();">
     <nav>
+    <div class="dropdown"style="float:right;">
+      <button class="dropbtn">Menu</button>
+      <div class="dropdown-content">
       <ul id="navigation">
         <li><a href="/">Home</a></li>
       </ul>
+      </div>
+    </div>
     </nav>
     
     <h1 id="page-title">User Page</h1>
-    <div id="about-me-container">Loading...</div>
+    <div class="about-me-container" id="about-me-container">Loading...</div>
   <div id="about-me-form" class="hidden">
   <form action="/about" method="POST">
-    <textarea name="about-me" placeholder="about me" rows=4 required></textarea>
+    <textarea class= "about-me-input" id= "about-me-input" name="about-me" placeholder="Say something about yourself..." rows=1 required></textarea>
```

Comment:
same w/ class and id names here?

Reply:
Got rid of id's, couldn't use them in CSS anyways

Reply:
So I put some back, because they were being used for certain parts of the code, but I did it to try and improve organization